src/components/map/map-control.js

```diff
@@ -162,12 +187,109 @@ const MapLegendPanel = ({layers, isActive, scale, toggleMenuPanel, isExport}) =>
     <MapControlPanel
       scale={scale}
       header={'Layer Legend'}
-      onClick={toggleMenuPanel}
+      onClick={onToggleMenuPanel}
       isExport={isExport}
     >
       <MapLegend layers={layers}/>
     </MapControlPanel>
+);
+
+MapLegendPanel.displayName = 'MapControlPanel';
+
+const SplitMapButton = React.memo(({isSplit, mapIndex, onToggleSplitMap}) => (
+  <MapControlButton
+    active={isSplit}
+    onClick={e => {
+      e.preventDefault();
+      onToggleSplitMap(isSplit ? mapIndex : undefined);
+    }}
+    key={`split-${isSplit}`}
+    className="map-control-button split-map"
+    data-tip
+    data-for="action-toggle"
+  >
+    {isSplit ? <Delete height="18px" /> : <Split height="18px" />}
+    <MapControlTooltip
+      id="action-toggle"
+      message={
+        isSplit ? 'Close current panel' : 'Switch to dual map view'
+      }
+    />
+  </MapControlButton>
+));
+
+SplitMapButton.displayName = 'SplitMapButton';
+
+const Toggle3dButton = React.memo(({dragRotate, onTogglePerspective}) => (
+  <MapControlButton
+    onClick={e => {
+      e.preventDefault();
+      onTogglePerspective();
+    }}
+    active={dragRotate}
+    data-tip
+    data-for="action-3d"
+  >
+    <Cube3d height="22px" />
+    <MapControlTooltip
+      id="action-3d"
+      message={dragRotate ? 'Disable 3D Map' : '3D Map'}
+    />
+  </MapControlButton>
+));
+
+Toggle3dButton.displayName = 'Toggle3dButton';
+
+const StyledToolBar = styled(Toolbar)`
+  position: absolute;
+  right: 32px;
+`;
+
+const MapDrawPanel = React.memo(({editor, isActive, onToggleMenuPanel, onSetEditorMode}) => {
+  return (
+    <div style={{position: 'relative'}}>
+      {isActive ? (
+        <StyledToolBar show={isActive} direction="column">
+          <ToolbarItem
```

Comment:
Now that clicking on polygon automatically selectes it. Do we still need this to manually enter selection mode?

Reply:
We need this for two reasons:
- enable drawing on the map
- the selection of a polygon only works when we are in editor mode. when we turn editor mode off we are not able to select polygons by clicking on it